ProjectEureka/project-eureka-flutter
# PE-105 | Changed App Icon and Name

What I Did: 

- Changed the default flutter icon and name to EureQa. 

How to Test: 

- Uninstall and reinstall the app. 
- Run flutter pub get. 
- Once the app is loaded, check to see if the app icon and name have been changed to our logo and name.
